src/Avalonia.Controls/ProgressRing.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using Avalonia.Controls.Metadata;
+using Avalonia.Controls.Primitives;
+using Avalonia.Layout;
+using Avalonia.Media;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// A control used to indicate the progress of an operation.
+    /// </summary>
+    [PseudoClasses(":preserveaspect", ":indeterminate")]
```

Comment:
Not sure if ":preserveaspect" is useful pseudoclass. Developer can always use attribute selector ```ProgressRing[IsIndeterminate=True]```
"indeterminate" can be here for consistency with progressbar though.

Reply:
@maxkatz6 ...so does that mean to remove it completely, then? ~~tbh I was never entirely sure why *any* boolean properties had corresponding pseudoclasses, but yeah sure I'll get rid of that one if that's what you're saying to do~~

Reply:
Yes, I was talking about removing it.
Some pseudoclasses were added to match HTML CSS pseudoclasses, some other were added because they are commonly used (:pointerover).

Reply:
ok yeah I'll get rid of that now